DOCUMENTATION.md

```diff
@@ -512,4 +514,23 @@ You can then update the property you want to update using the `set` command:
 
 ```content-cli set connection --profile <profile> --dataPoolId <dataPoolId> --connectionId <connectionId> --property <property> --value <value>```
 
+### Config commands
+
+
+### List all packages across flavors
+
+With this command you can retrieve a list of all packages across flavors within a team.
+The command takes your permissions into consideration and only lists the
+packages you have access to.
+To list only packages of certain flavors you can use the `--flavors` option and pass a list of flavors.
+
+-   It is also possible to download packages in JSON format by adding '--json' option.
+-   When the JSON format option is used, it is also possible to include package dependencies by adding '--withDependencies' flag.
+-   When the JSON format option is used, it is also possible to filter packages by adding '--packageKeys' parameter
+
+```
+content-cli config list -p <your-chosen-profile>
+content-cli config list -p <profileName> --json --packageKeys <package1> <package2> --flavors <flavor1> <flavor2>
```

Comment:
It seems that on the actual usage, flavors are comma-separated. Should we specify that on the documentation as well?

Reply:
Using comma-separated values might work because it is treating the whole string as an entry, but we want users to abide by the documentation to match the intended use.

Reply:
Just to confirm, this works both ways right?

1. `--flavors studio process-workspace`
2. `--flavors studio,process-workspace`

Reply:
Yes, that is correct.